pymatgen/entries/corrections_calc.py

```diff
@@ -67,24 +67,22 @@ class CorrectionCalculator:
         "H",
     ]  # species that we're fitting corrections for
 
-    def __init__(self, exp_gz: str, comp_gz: str) -> None:
+    def __init__(self, max_error: float = 0.1, allow_unstable: bool = False, allow_polyanions: bool = False) -> None:
 
         """
         Initializes a CorrectionCalculator.
 
         Args:
-            exp_gz: name of gzip file that contains experimental data
-                    data in gzip file should be a list of dictionary objects with the following keys/values:
-                    {"formula": chemical formula, "exp energy": formation energy in eV/formula unit,
-                    "uncertainty": uncertainty in formation energy, "warnings": {"polyanion": str describing
-                    polyanions in compound or None, "large_uncertainty": str describing if compound has a large
-                    uncertainty value or None, "unstable": str describing if compound has high e above hull or None}}
-            comp_gz: name of gzip file that contains computed entries
-                    data in gzip file should be a dictionary of {chemical formula: ComputedEntry}
+            max_error: maximum tolerable relative uncertainty in experimental energy
+            allow_unstable: whether unstable entries are to be included in the fit
```

Comment:
Clarify "unstable" - "whether unstable entries (i.e., e_above_hull > 0) are included in the fit"